Docs/developer_guide/script_repository.md

```diff
@@ -8,6 +8,15 @@ To run a Python code snippet automatically at each application startup, add it t
 
 :::
 
+:::{note}
+
+**Fully worked examples**:
+* The [Slicer source code](https://github.com/Slicer/Slicer) is about half written in Python and can be used to find working examples
```

Comment:
```suggestion
The Slicer source code has Python [scripted modules](https://github.com/Slicer/Slicer/tree/master/Modules/Scripted) and [scripted Segmentation Editor effects](https://github.com/Slicer/Slicer/tree/master/Modules/Loadable/Segmentations/EditorEffects/Python) that can be used as working examples.
```

It turns that ~10% of Slicer is written in python:
![image](https://user-images.githubusercontent.com/219043/125210291-e3bb9980-e26c-11eb-89ac-0fcf91d298c2.png)


```
Modules/Scripted/DICOMPlugins
Modules/Scripted/DICOMLib
Modules/Scripted/SampleData
Modules/Scripted/DICOM
Modules/Scripted/SegmentEditor
Modules/Scripted/SegmentStatistics
Modules/Scripted/Endoscopy
Modules/Scripted/DICOMPatcher
Modules/Scripted/ScreenCapture
Modules/Scripted/CropVolumeSequence
Modules/Scripted/DataProbe
Modules/Scripted/LabelStatistics
Modules/Scripted/VectorToScalarVolume
```